tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.ps1

```diff
@@ -0,0 +1,28 @@
+$ErrorActionPreference = 'Stop'
+Set-PSDebug -trace 1
+$subscriptionId = "2cd617ea-1866-46b1-90e3-fffb087ebf9b"
+$env:AZURE_STORAGE_ACCOUNT="stresstestcharts"
+$env:AZURE_STORAGE_KEY=$(az storage account keys list --subscription $subscriptionId --account-name $env:AZURE_STORAGE_ACCOUNT -o tsv --query '[0].value')
+if($LASTEXITCODE) {exit $LASTEXITCODE}
+
+Remove-Item -Force ./*.tgz
+if($LASTEXITCODE) {exit $LASTEXITCODE}
+
+helm package  .
+if($LASTEXITCODE) {exit $LASTEXITCODE}
+helm repo index --url https://stresstestcharts.blob.core.windows.net/helm/ --merge index.yaml .
+if($LASTEXITCODE) {exit $LASTEXITCODE}
+
+# The index.yaml in git should be synced with the index.yaml already in blob storage
+# az storage blob download -c helm -n index.yaml -f index.yaml
+
+$files = (Get-Item *.tgz).Name
```

Comment:
I think it's maybe worth adding a terminal prompt here (e.g. `Read-Host` or something) so you can double check what's generated before doing the upload. This is something I wanted but hadn't added.